internal/ip/iplayer_linux.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
Since this is really the equivalent of `ip_darwin.go` in terms of functions included, I would call this file `ip_linux.go` and the other linux file I would call it `ip_linux_bpf.go` or `ip_linux_WHATEVER.go` so that it follows alphabetically the original file.